### ISSUE TITLE
The GPU clock speed needs to be manually set to 1600 MHz for optimal performance.

### ISSUE THREAD

**Issue by jams3223** [ValveSoftware/SteamOS#1279]
#### Your system information

* SteamOS version:  3.5.7
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
Every game that I play never fully utilizes the GPU unless I manually clock it to 1600 MHz, which greatly improves performance and makes full use of it. It happens on every game, and some people who don't use the Steam overlay might not notice anything since they don't know they're playing at a lower fps.

#### Steps for reproducing this issue:

1. Turn the Steam Deck on in Game Mode on a Steam Deck OLED
2. Start any game with the default clock speed setting unchanged
4. Watch performance degrades with low frame pacing
5. Manually clock the GPU to 1600 MHz to improve the performance
6. The GPU will be fully utilized

**Comment by waspennator** [comment]
Is there a game you best see this behavior in action outside of just "every game", mainly asking since it helps narrow things down. Also since another issue report got closed cause they didn't state what games they tested it out in and only said "every game"

**Comment by jams3223** [comment]
> Is there a game you best see this behavior in action outside of just "every game", mainly asking since it helps narrow things down. Also since another issue report got closed cause they didn't state what games they tested it out in and only said "every game"

Known list of games affected:
American Truck Simulator
Cyberpunk 2077
Star Wars Jedi: Fallen Order
Signalis
Valheim

2D games tend to be less noticeable since they use a lot less resources.

**Comment by YoreonX** [comment]
> #### Your system information
> 
> * SteamOS version:  3.5.7
> * Opted into Steam client beta?: No
> * Opted into SteamOS beta?: No
> * Have you checked for updates in Settings > System?: Yes
> 
> #### Please describe your issue in as much detail as possible:
> Every game that I play never fully utilizes the GPU unless I manually clock it to 1600 MHz, which greatly improves performance and makes full use of it. It happens on every game, and some people who don't use the Steam overlay might not notice anything since they don't know they're playing at a lower fps.
> 
> #### Steps for reproducing this issue:
> 
> 1. Turn the Steam Deck on in Game Mode on a Steam Deck OLED
> 2. Start any game with the default clock speed setting unchanged
> 4. Watch performance degrades with low frame pacing
> 5. Manually clock the GPU to 1600 MHz to improve the performance
> 6. The GPU will be fully utilized
> 

Join the Preview channel and update to 3.5.8 and tell me if the issue persist.

**Comment by Oliver-1xok** [comment]
> 4\. Manually clock the GPU to 1600 MHz to improve the performance

Of course, this will add a few frames here and there because the clock doesn't have to be ramped up, but you'll also have a much shorter battery life in many games.

I played Wolfenstein 2 yesterday. Played at 720p and maximum settings. That often pushes the GPU to the limit. Nevertheless, I didn't notice any problems on the performance side. I had an almost constant 60 FPS.

**Comment by Glorydayreject** [comment]
@Oliver-1xok 
I can get consistent frames too, but that doesn't stop the game from stuttering. I made a post about this exact issue here 

https://github.com/ValveSoftware/SteamOS/issues/1266#issue-2019458233
